Makefile

```diff
@@ -103,12 +103,13 @@ generate-deepcopy:
 	controller-gen object paths=api/v1/nodecrd/node_types.go paths=api/v1/nodecrd/groupversion_info.go  output:dir=api/v1/nodecrd
 
 generate-crds:
-    # Generate CRDs based on Volume and AvailableCapacity type and group info
```

Comment:
What problem are you solving here?

Reply:
It makes easier to copy crds into csi-baremetal-operator chart and give the opportunity to check in pr-validation that we have actual files in our repo (yamls in commit equals to `make generate-crds` output). Also we don't support these charts, it was confusing for me

Reply:
I recommend you to not mix 2 different tasks in single pr. Please open a separate issue for charts modifications

Reply:
Changes were removed from this pr, I will open another